src/vm/stacksampler.h

```diff
@@ -55,17 +54,15 @@ class StackSampler
     typedef MapSHash<MethodDesc*, CountInfo> CountInfoHash;
     typedef CountInfoHash::element_t CountInfoHashEntry;
 
-    typedef MapSHash<MethodDesc*, ADID> JitInfoHash;
```

Comment:
That the only instance of `JitInfoHash`, the `m_jitInfo`, is not used for anything else than adding an entry into it. The only place where it was used for look up (`StackSampler::GetDomainId`) was removed. So it seems we can remove the typedef and the instance.